pilot/pkg/model/sidecar.go

```diff
@@ -213,6 +213,7 @@ func convertIstioListenerToWrapper(ps *PushContext, sidecarConfig *Config,
 	}
 
 	if istioListener.Hosts != nil {
+		// TODO: BUG, this means you can't have 2 explicit imports for same namespace ( override each other )
```

Comment:
yes. you need a `map[namespace]=[]Hostname`

Reply:
I'll leave it to you or separate PR - the goal for this PR is to unblock the testing, don't want to mix it with changing the struct.